egs/librimix/ConvTasNet/local/prepare_data.sh

```diff
@@ -17,3 +17,9 @@ cd LibriMix
 cd $current_dir
 $python_path local/create_local_metadata.py --librimix_dir $storage_dir/Libri$n_src"Mix"
 
+
+# TODO: check folders and outdir
```

Comment:
Here, we retrieve the txt files for each split and gather them in .csv files that we'll store in the local `data` folder.